### PR TITLE
i18n: Mark strings for translation

### PR DESCRIPTION
**Pull request by tombettany**
In preparation for i18n, mark all the appropriate strings for
translation.

### CODE REVIEW

**Comment by tombettany** [comment]
@convolu Could you take a look at this?

File: bin/kano-init-flow [hunk 1]
```diff
@@ -17,83 +17,43 @@
...
-            self.resize(590, 100)
 
-        def exit_flow(self):
-            if self.need_reboot:
-                self.clear_win()
-                RebootScreen(self)
-            else:
-                # Init flow completed
-                sys.exit(0)
+    enforce_root(_('Please run it as root / sudo user!'))
```
Comment (convolu) [inline bin/kano-init-flow]: We need to make sure that `enforce_root` supports unicode

Comment (tombettany) [inline bin/kano-init-flow]: I'm going through toolset right now to ensure things like that. Good spot.

Comment (tombettany) [inline bin/kano-init-flow]: I've just checked [`enforce_root`](https://github.com/KanoComputing/kano-toolset/blob/d12e1b60c6c0e852f41932561f3043bf311a961d/kano/utils.py#L351-L354) in toolset. It does

```
sys.stderr.write(msg + "\n")
```

which, from a quick test, seems fine with unicode.

File: kano_init_flow/audio_screen.py [hunk 2]
```diff
@@ -184,11 +185,11 @@ def __init__(self, win):
 
         self.template = Template(
             img_path=os.path.join(MEDIA_DIR, "Audio_See_the_light.png"),
-            title="Can you see the light?",
-            description="If the power plugs are connected correctly, " \
-                        "you should see a blue light.",
-            button1_text="YES",
-            button2_text="NO"
+            title=_("Can you see the light?"),
+            description=_("If the power plugs are connected correctly, " \
```
Comment (convolu) [inline kano_init_flow/audio_screen.py]: I think that the backslash is not needed when a line is broken within a parenthesis, but it shouldn't be an issue

File: kano_init_flow/first_screen.py [hunk 3]
```diff
@@ -28,9 +28,10 @@ def __init__(self, win):
         self.win = win
         self.template = Template(
             img_path=os.path.join(MEDIA_DIR, "made_it.png"),
-            title="We made it!",
-            description="You made a computer - now let's give it new powers!",
-            button1_text="START SETUP"
+            title=_("We made it!"),
+            description=_("You made a computer - " \
```
Comment (convolu) [inline kano_init_flow/first_screen.py]: Same here (the backslash)

File: kano_init_flow/internet_screen.py [hunk 4]
```diff
@@ -93,11 +93,11 @@ def __init__(self, win):
 
         self.template = Template(
             img_path=os.path.join(MEDIA_DIR, "no_internet.png"),
-            title="No internet?",
-            description="Try again, or connect later. You need internet " \
-                        "for most of Kano's coolest powers.",
-            button1_text="TRY AGAIN",
-            orange_button_text="Connect later"
+            title=_("No internet?"),
+            description=_("Try again, or connect later. You need internet " \
```
Comment (convolu) [inline kano_init_flow/internet_screen.py]: Here as well (backslash)

File: kano_init_flow/internet_screen.py [hunk 5]
```diff
@@ -145,11 +145,11 @@ def __init__(self, win):
 
         self.template = Template(
             img_path=os.path.join(MEDIA_DIR, "internet_trouble.png"),
-            title="Internet trouble? We can help!",
-            description="Visit http://help.kano.me on another device, " \
-                        "or email wifi@kano.me. You can play offline in " \
-                        "the meantime.",
-            button1_text="PLAY OFFLINE"
+            title=_("Internet trouble? We can help!"),
+            description=_("Visit http://help.kano.me on another device, " \
```
Comment (convolu) [inline kano_init_flow/internet_screen.py]: Here too (backslash)

**Comment by convolu** [comment]
Other than the minor points mentioned above (you don't have to fix the backslashes, I just pointed it out) it looks good to me :shipit:

**Comment by tombettany** [comment]
@convolu Good spot, I've fixed the trailing backslashes and will merge now.

**Pull request merged**